pkg/handler/validate.go

```diff
@@ -0,0 +1,18 @@
+package handler
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+func (h *Handler) Validate(c *gin.Context) {
+	err := h.sealer.Validate(c.Request.Body)
```

Comment:
could you forward the context c to Validate and use it instead of context.TODO() there?